### PR TITLE
Features/validate alt contact #162623064

### PR DESCRIPTION
**Pull request by chrisdolendo**

### CODE REVIEW

**Comment by akegan** [comment]
@chrisdolendo did you find a solution on this?

File: app/javascript/components/applications/application_form/AlternateContactSection.js [hunk 1]
```diff
@@ -1,23 +1,39 @@
...
+import validate from '~/utils/form/validations'
+import { Field } from '~/utils/form/Field'
 
 const {
   alternateContactOptions,
   phoneTypeOptions
 } = formOptions
 
+const validateError = (formValues) => {
+  return {
```
Comment (akegan) [inline app/javascript/components/applications/application_form/AlternateContactSection.js:23]: the way we do it for householdMember validations, is to check if it's empty before doing the validations. 
https://github.com/Exygy/sf-dahlia-lap/blob/master/app/javascript/components/applications/application_form/PaperApplicationForm.js#L63-L74. I think if you did a similar thing here that would get rid of the need for `validateFirstLastName`

**Review by akegan — DISMISSED** [review]
just one comment/request. Otherwise code looks good. I'll QA now

File: app/javascript/components/applications/application_form/AlternateContactSection.js (hunk 1 above)
Comment (chrisdolendo) [reply app/javascript/components/applications/application_form/AlternateContactSection.js:23]: Yes, I initially did this but the formApi wouldn’t get updated when I tried to clear the empty string. I can try again and get a 2nd pair of eyes on it

Comment (chrisdolendo) [reply app/javascript/components/applications/application_form/AlternateContactSection.js:23]: @akegan I implemented this way and looks like it works -- modifying formValues directly than accessing the formApi :+1:

**Comment by chrisdolendo** [comment]
@akegan ready for re-review!

File: app/javascript/components/applications/application_form/AlternateContactSection.js (hunk 1 above)
Comment (akegan) [reply app/javascript/components/applications/application_form/AlternateContactSection.js:23]: hm, for this I meant that you could replace validateFirstLastName with just validate.isPresent. Let me know if you want to hop on a quick call about it

**Review by akegan — DISMISSED** [review]
I put this in slack too but want to reiterate here, that right now if I add an alternate contact to a paper app, save, then go to edit it, I can't successfully remove the alternate contact

I get these validation errors
image

**Review by akegan — APPROVED** [review]
Looks good! did a bunch of editing and adding and taking away alt contacts and it looks good

**Pull request merged**